doc/tutorial/tutorial.md

```diff
@@ -1026,18 +1026,18 @@ typedef option_ip4_pkt_t = IP4Some{ p: ip4_pkt_t }
 
 However, defining an extra variant type for each type in the program quickly becomes a burden on
 the programmer.  Fortunately, DDlog allows us to define a generic option type that can be
-instantiated for any concrete type:
+instantiated for any concrete type.  This types is defined in the DDlog standard library as follows:
```

Comment:
It's not because of performance (performance might actually be better with `Option`), but simplicity. Treating some of the sets and Option<T> instead of Set<T> would introduce a bunch of new cases in the already complex OVSDB compiler.